cypress/integration/executions_spec.js

```diff
@@ -158,5 +158,59 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show an execution with limited information', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = 'b313e777-d28a-435b-a0dd-f1fad08116t1';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestAndPublishGranuleStateMachine-yCAhWOss5Xgo:b313e777-d28a-435b-a0dd-f1fad08116t1';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestAndPublishGranuleStateMachine-yCAhWOss5Xgo';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
```

Comment:
Is there any reason to not do `cy.visit` directly to the URL for the execution? I think we already have a test that does a click on a row in the execution list, so we don't need to re-test that